crates/cdk/src/wallet/swap.rs

```diff
@@ -188,13 +199,16 @@ impl Wallet {
         spending_conditions: Option<SpendingConditions>,
         include_fees: bool,
     ) -> Result<PreSwap, Error> {
+        tracing::info!("Creating swap");
         let active_keyset_id = self.get_active_mint_keyset().await?.id;
 
         // Desired amount is either amount passed or value of all proof
         let proofs_total = proofs.total_amount()?;
 
         let ys: Vec<PublicKey> = proofs.ys()?;
-        self.localstore.set_pending_proofs(ys).await?;
+        self.localstore
+            .update_proofs_state(ys, State::Pending)
+            .await?;
```

Comment:
I think we should use `Reserved` here, and keep `Pending` as a state used by the mint ie in a melt operation where the ln payment is taking some time.